devops/tasks.py

```diff
@@ -19,11 +19,14 @@ def generate_release_id() -> str:
     return "".join(random.choice(chars) for _ in range(length))  # nosec
 
 
-def build_images(ctx, components, dry_run=False):
-    big_label(logger.info, "Building images")
+def build_images(ctx, components, dry_run=False, docker_args=None):
+    if docker_args is None:
+        docker_args = []
+
+    big_label(logger.info, f"Building images with args {docker_args}")
```

Comment:
You likely want to make this now conditional based on if there were args or not, and use a `"--build-arg ".join(docker_args)` or some such